Library/Homebrew/dev-cmd/extract.rb

```diff
@@ -113,7 +113,7 @@ def self.extract
     end
     destination_tap.install unless destination_tap.installed?
 
-    repo = source_tap.path
+    repo = source_tap.path.pathname
```

Comment:
I'm sorry this is taking so much back and forth! 

This now looks pretty weird?

I'd expect that `source_tap.path` returns a `Pathname`. If not, maybe something like

```suggestion
    repo = source_tap.git_repo.path
```

or 

```suggestion
    repo = source_tap.git_repo_path.pathname
```

would make more sense?